mods/seasonal/moves.js

```diff
@@ -518,9 +536,10 @@ exports.BattleMovedex = {
 	// panpawn
 	lafireblaze420: {
 		accuracy: 75,
-		basePower: 150,
+		basePower: 0,
+		damage: 150,
 		category: "Physical",
-		shortDesc: "20% chance to burn the target.",
+		shortDesc: "Does 150 damage. 20% chance to burn the target.",
```

Comment:
i thought this move was supposed to have 150 base power? not set damage

Reply:
In the bug reports issue, I read that it's 150 damage

Reply:
I have asked panpawn and  he confirmed 150 damage